helm/linkerd2-app/charts/linkerd-crds/templates/serviceprofile.yaml

```diff
@@ -9,8 +9,8 @@ metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
   labels:
-    linkerd.io/control-plane-ns: {{.Values.namespace}}
-    {{- include "linkerd.giantswarm-labels" . | nindent 4 }}
```

Comment:
what your reason for removing the "giantswarm-labels". Labeling all the components helps people on support and in some cases is required for observability.

Reply:
nice catch. I need to fix it. with the charts rename from upstream I missed it. thanks

Reply:
fixed